src/win32/mouse.cpp

```diff
@@ -14,6 +14,10 @@ Point GetPosition() {
 void MoveAbsolute(int x, int y) {
     // (0, 0) is the top left corner
     // (65535, 65535) is the bottom right corner
+
+    x = x * 65535 / GetDeviceCaps(GetDC(NULL), HORZRES);
```

Comment:
As there is a +1 or -1 rounding error, this should be converted to a float or double and rounded (`std::round`).